--- mcp/vanillaSrc/minecraft_server/net/minecraft/server/dedicated/DedicatedServer.java
+++ mcp/src/minecraft_server/net/minecraft/server/dedicated/DedicatedServer.java
@@ -13,10 +13,13 @@
 import java.util.Collections;
 import java.util.List;
 import java.util.Random;
 import java.util.concurrent.TimeUnit;
 import java.util.regex.Pattern;
+
+import io.minetweak.Minetweak;
+import io.minetweak.ServerLoadedEvent;
 import net.minecraft.command.ICommandSender;
 import net.minecraft.crash.CrashReport;
 import net.minecraft.crash.ICrashReportDetail;
 import net.minecraft.entity.player.EntityPlayer;
 import net.minecraft.network.rcon.IServer;
@@ -207,10 +210,13 @@
             LOGGER.info("Preparing level \"{}\"", new Object[]{this.getFolderName()});
             this.loadAllWorlds(this.getFolderName(), this.getFolderName(), k, worldtype, s2);
             long i1 = System.nanoTime() - j;
             String s3 = String.format("%.3fs", new Object[]{Double.valueOf((double)i1 / 1.0E9D)});
             LOGGER.info("Done ({})! For help, type \"help\" or \"?\"", new Object[]{s3});
+
+            Minetweak.getInstance().getEventBus().post(new ServerLoadedEvent());
+            
             if(this.settings.getBooleanProperty("enable-query", false)) {
                LOGGER.info("Starting GS4 status listener");
                this.theRConThreadQuery = new RConThreadQuery(this);
                this.theRConThreadQuery.startThread();
             }
