--- mcp/vanillaSrc/minecraft_server/net/minecraft/server/MinecraftServer.java
+++ mcp/src/minecraft_server/net/minecraft/server/MinecraftServer.java
@@ -735,11 +735,11 @@
          LOG.info(msg);
       }
    }
 
    public String getServerModName() {
-      return "vanilla";
+      return "minetweak";
    }
 
    public CrashReport addServerInfoToCrashReport(CrashReport report) {
       report.getCategory().func_189529_a("Profiler Position", new ICrashReportDetail<String>() {
          public String call() throws Exception {
